.github/workflows/fmu-tools.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['2.7', '3.6', '3.7', '3.8']
+        python-version: ['3.6', '3.7', '3.8']
```

Comment:
Should we skip 2.7 now? It makes life easier...

Reply:
Yes, lets make life easier.